src/providers/song-info-front.ts

```diff
@@ -200,6 +200,11 @@ export default (api: YoutubePlayer) => {
     for (const status of ['playing', 'pause'] as const) {
       video.addEventListener(status, playPausedHandlers[status]);
     }
+
+    const data = api.getVideoData();
```

Comment:
It would only make a duplicate request on startup if we used youtubei.js to fetch the data.

Can't imagine it being flagged.

Reply:
Made it a bit more reliable, it now attempts to grab the album name.
It will fail to do so if an ad is playing at that exact moment, but the album name will simply be undefined in that scenario.

Reply:
eureka!